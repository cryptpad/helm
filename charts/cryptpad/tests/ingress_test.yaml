suite: test ingress
templates:
  - ingress.yaml
  - service.yaml
tests:
  - it: should set values properly
    set:
      annotations.kubernetes.io/ingress.class: nginx
      service.type: ClusterIP
      service.externalPort: 80
      service.internalPort: 8080
      service.name: tcp
      ingress:
        enabled: true
        hosts:
        - host: localhost
          paths:
            - path: /
              pathType: ImplementationSpecific
    release:
      name: release
    asserts:
      - isKind:
          of: Ingress
        template: ingress.yaml
      - equal:
          path: metadata.name
          value: release-cryptpad
      - matchSnapshot:
          path: spec
        template: ingress.yaml
