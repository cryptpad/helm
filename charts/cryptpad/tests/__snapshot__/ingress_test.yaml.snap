should set values properly:
  1: |
    rules:
      - host: localhost
        http:
          paths:
            - backend:
                service:
                  name: release-cryptpad
                  port:
                    number: null
              path: /
              pathType: ImplementationSpecific
